### ISSUE TITLE
Bad argument #2 to 'sort ' (function expected, got boolean)

### ISSUE THREAD

**Issue by olimorris** [noib3/cokeline.nvim#24]
The latest commits seem to be throwing an error for me:

```lua
Error detected while processing VimEnter Autocommands for "*":
E5108: Error executing lua ...pack/packer/start/cokeline.nvim/lua/cokeline/buffers.lua:245: bad argument #2 to 'sort
' (function expected, got boolean)
```
<details>
  <summary>My config is here</summary>

```lua
local M = {}

function M.setup()
	local present, bufferline = pcall(require, "cokeline")
	if not present then
		return
	end

	local spacer = "  "
	local colors = require("onedarkpro").get_colors(vim.g.onedarkpro_style)

	bufferline.setup({

		cycle_prev_next_mappings = true,
		show_if_buffers_are_at_least = 2,

		default_hl = {
			focused = {
				fg = colors.purple,
				bg = "NONE",
				style = "bold",
			},
			unfocused = {
				fg = colors.gray,
				bg = "NONE",
			},
		},
		buffers = {
			filter = function(buffer)
				return buffer.type ~= "terminal" and buffer.type ~= "quickfix"
			end,
		},
		rendering = {
			max_line_width = 25,
		},
		components = {
			{
				text = function(buffer)
					return buffer.index ~= 1 and spacer
				end,
			},
			{
				text = function(buffer)
					return buffer.index .. ": "
				end,
				hl = {
					style = function(buffer)
						return buffer.is_focused and "bold" or nil
					end,
				},
			},
			{
				text = function(buffer)
					return buffer.unique_prefix
				end,
				hl = {
					fg = function(buffer)
						return buffer.is_focused and colors.purple or colors.gray
					end,
					style = "italic",
				},
			},
			{
				text = function(buffer)
					return buffer.filename
				end,
				hl = {
					style = function(buffer)
						return buffer.is_focused and "bold" or nil
					end,
				},
			},
			{
				text = function(buffer)
					return buffer.is_modified and " ●"
				end,
				truncation = {
					priority = 1,
				},
				hl = {
					fg = function(buffer)
						return buffer.is_focused and colors.red
					end,
				},
			},
			{
				text = spacer,
			},
		},
	})
end

return M
```
</details>

Commit `176fe67c4f5139fc88c3185d012b6d9f2b64cee2` from 15 November worked absolutely fine.

**Comment by noib3** [comment]
For now you can fix it by passing `new_buffers_position = 'last'` to the `buffers` table. The problem is the user defined `buffers` table is overriding the default one instead of updating it.

**Comment by noib3** [comment]
This should be fixed by e64228871d1f3faa501570edc1166ffe04e9b571. Feel free to reopen if you still experience problems.
